app/controllers/api/v1/reviews_controller.rb

```diff
@@ -1,22 +1,35 @@
 class Api::V1::ReviewsController < ApplicationController
+
   def create
-    new_review = Review.new(review_params)
     wizard = Wizard.find(review_params[:wizard_id])
     if user_signed_in?
-      # Design descision to be made: will the react element need the wizard?
+      new_review = Review.new(review_params.merge({user_id: current_user.id}))
```

Comment:
I would not set the user this way, because it gets confused with the params hash. I would explicitly say on its own line:

```ruby
new_review.user = current_user
```